### PR TITLE
AWS terraform module - storage

### PR DESCRIPTION
**Pull request by TagadaPoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
...
+
 # terraform-aws-storage
+
 A terraform module for Quortex infrastructure AWS persistent storage layer.
+
+It provides a set of resources necessary to provision the bucket and access key on Amazon AWS.
+
+![infra_diagram]
+
+This module is available on [Terraform Registry][registry_tf_aws-eks_cluster].
```
Comment (vincentmrg) [inline README.md]:
```suggestion
This module is available on [Terraform Registry][registry_tf_aws-eks_storage].
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
...
+- as many buckets in Amazon S3 as defined in the configuration
+- a new user, with access to the bucket
+- the key ID and secret for this user
+
+
+## Usage example
+
+```
+module "storage" {
+  source = "git::https://github.com/quortex/terraform-aws-storage.git"
```
Comment (vincentmrg) [inline README.md]: Once published on public registry, it will be like that.

```suggestion
  source = "quortex/storage/aws"
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+
+- [terraform-aws-eks-load-balancer][registry_tf_aws-eks_load_balancer] - A terraform module for Quortex infrastructure AWS load balancing layer.
+
+- [terraform-aws-storage][registry_tf_aws-eks_storage] - A terraform module for Quortex infrastructure AWS persistent storage layer.
+
+## Help
+
+**Got a question?**
+
+File a GitHub [issue](https://github.com/quortex/terraform-aws-eks-storage/issues) or send us an [email][email].
```
Comment (vincentmrg) [inline README.md]:
```suggestion
File a GitHub [issue](https://github.com/quortex/terraform-aws-storage/issues) or send us an [email][email].
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,72 @@
...
+
+
+  [logo]: https://storage.googleapis.com/quortex-assets/logo.webp
+  [infra_diagram]: https://storage.googleapis.com/quortex-assets/infra_aws_001.jpg
+
+  [email]: mailto:info@quortex.io
+
+  [registry_tf_modules]: https://registry.terraform.io/modules/quortex
+  [registry_tf_aws-eks_network]: https://registry.terraform.io/modules/quortex/network/aws-eks
```
Comment (vincentmrg) [inline README.md]: The provider will be "aws" on terraform registry

```suggestion
  [registry_tf_aws-eks_network]: https://registry.terraform.io/modules/quortex/network/aws
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,72 @@
...
+  [registry_tf_aws-eks_cluster]: https://registry.terraform.io/modules/quortex/eks-cluster/aws-eks
```
Comment (vincentmrg) [inline README.md]: The provider will be "aws" on terraform registry

```suggestion
  [registry_tf_aws-eks_cluster]: https://registry.terraform.io/modules/quortex/eks-cluster/aws
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,72 @@
...
+  [registry_tf_aws-eks_load_balancer]: https://registry.terraform.io/modules/quortex/load-balancer/aws-eks
```
Comment (vincentmrg) [inline README.md]: The provider will be "aws" on terraform registry

```suggestion
  [registry_tf_aws-eks_load_balancer]: https://registry.terraform.io/modules/quortex/load-balancer/aws
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,72 @@
...
+  [registry_tf_aws-eks_storage]: https://registry.terraform.io/modules/quortex/storage/aws-eks
```
Comment (vincentmrg) [inline README.md]: The provider will be "aws" on terraform registry

```suggestion
  [registry_tf_aws-eks_storage]: https://registry.terraform.io/modules/quortex/storage/aws
```

File: versions.tf [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = "~> 0.12.6"
```
Comment (vincentmrg) [inline versions.tf]:
```suggestion
  required_version = ">= 0.12"
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,72 @@
...
+
+Check out these related projects.
+
+- [terraform-aws-network][registry_tf_aws-eks_network] - A terraform module for Quortex infrastructure network layer.
+
+- [terraform-aws-eks-cluster][registry_tf_aws-eks_cluster] - A terraform module for Quortex infrastructure AWS cluster layer.
+
+- [terraform-aws-eks-load-balancer][registry_tf_aws-eks_load_balancer] - A terraform module for Quortex infrastructure AWS load balancing layer.
+
+- [terraform-aws-storage][registry_tf_aws-eks_storage] - A terraform module for Quortex infrastructure AWS persistent storage layer.
```
Comment (vincentmrg) [inline README.md]: Not essential, this is the current module

Comment (TagadaPoe) [reply README.md]: Changed as suggested

**Pull request merged**